dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry10/internal/rest/TelemetryContainerFilter.java

```diff
@@ -173,6 +173,7 @@ public void filter(final ContainerRequestContext request) {
                     }
 
                     currentSpan.setAttribute(SemanticAttributes.HTTP_ROUTE, route);
+                    currentSpan.updateName(request.getMethod() + route);
```

Comment:
Does this pass all the existing tests? I'd have expected that you'd need to add a space between the method and the route.